samples/block_progress_bar.cpp

```diff
@@ -17,7 +17,7 @@ int main() {
   auto progress = 0.0f;
```

Comment:
Shouldn't `progress` now be of `size_t` type here?

Reply:
Good catch, fixed.